samples/app/MeshAssimp.cpp

```diff
@@ -97,7 +100,7 @@ uint64_t hashMaterialConfig(MaterialConfig config) {
     appendBooleanToBitMask(bitmask, config.alphaMode == AlphaMode::OPAQUE);
     appendBooleanToBitMask(bitmask, config.alphaMode == AlphaMode::MASKED);
     appendBooleanToBitMask(bitmask, config.alphaMode == AlphaMode::TRANSPARENT);
-    appendBooleanToBitMask(bitmask, config.numUVs == 1);
+    appendBooleanToBitMask(bitmask, config.maxUVIndex() == 1);
```

Comment:
You don't even need this in the key, you have the UV indices. Since maxUVIndex() is computed from the other fields, this is not useful in the key.